index.js

```diff
@@ -1 +1,15 @@
-module.exports = require('./elliptic')
+let BigIntSupportChecked = false
+let secp256k1
+
+if (!BigIntSupportChecked) {
```

Comment:
btw, why do we need this `if` wrapper?. We never check `BigIntSupportChecked` after we set it

Reply:
I think that's some leftover.